stage1/init/kvm/resources.go

```diff
@@ -66,3 +66,17 @@ func GetAppsResources(apps schema.AppList) (totalCpus, totalMem int64) {
 
 	return totalCpus, totalMem
 }
+
+// GetAppCapabilities is a filter which returns list of valid Linux capabilities
+// Function requires a manifest of application
+func GetAppCapabilities(app *types.App) []types.LinuxCapability {
+
+	var caps []types.LinuxCapability
+
+	for _, isolator := range app.Isolators {
+		if capSet, ok := isolator.Value().(types.LinuxCapabilitiesSet); ok {
+			caps = append(caps, capSet.Set()...)
+		}
+	}
+	return caps
+}
```

Comment:
This commit unnecessarily changes the permission of the source file: 100644 → 100755